### PR TITLE
wetransfer-upload v0.0.2

### PR DESCRIPTION
**Pull request by kawaiseb**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by viktorbenei** [comment]
Thanks @kawaiseb for the PR, we'll check it ASAP! ;)

File: steps/wetransfer-upload/v0.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+title: wetransfer-upload
```
Comment (godrei) [inline steps/wetransfer-upload/v0.0.2/step.yml]: step's title will appear on the workflow editor UI, please use: `WeTransfer` as title instead

File: steps/wetransfer-upload/v0.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- example_step_input: Default Value - you can leave this empty if you want to
```
Comment (godrei) [inline steps/wetransfer-upload/v0.0.2/step.yml]: Do not leave template/unused code in the step.yml please

File: steps/wetransfer-upload/v0.0.2/step.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      Description of this input.
+
+      Can be Markdown formatted text.
+    is_expand: true
+    is_required: true
+    summary: Summary. No more than 2-3 sentences.
+    title: Example Step Input
+    value_options: []
+outputs:
+- EXAMPLE_STEP_OUTPUT: null
```
Comment (godrei) [inline steps/wetransfer-upload/v0.0.2/step.yml]: Do not leave template/unused code in the step.yml please

File: steps/wetransfer-upload/v0.0.2/step.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+title: wetransfer-upload
+summary: |
+  Upload files to wetransfer and share a short url to download
+description: |
+  This step send files to wetransfer. Wetransfer give you a short url to download files. Limit 2GO.
```
Comment (godrei) [inline steps/wetransfer-upload/v0.0.2/step.yml]: Please describe more what this step is supposed to do, and what setup is required for using this step.

**Comment by godrei** [comment]
Hi @kawaiseb ,
are you sure, that this step can upload files to wetransfer?
Based on your step's source code (commit: 8d4a75db30e31408f00577bc867f62cdfadaff3f): your step's toolkit is: bash with entry point: step.sh.
However your step.sh file does not do anything expect the template commands.

**Comment by viktorbenei** [comment]
Indeed, @godrei is right.

I'd suggest @kawaiseb to try the step in your bitrise workflow; you can do that even before sharing it in the StepLib! Related docs: http://devcenter.bitrise.io/bitrise-cli/steps/#step-referenceid-format

P.S.: the "git clone step ref" for your step:

```
- git::https://github.com/kawaiseb/bitrise-step-wetransfer-upload.git@master: {}
```

File: steps/wetransfer-upload/v0.0.2/step.yml (hunk 2 above)
Comment (kawaiseb) [reply steps/wetransfer-upload/v0.0.2/step.yml]: Ok i will clean the step

File: steps/wetransfer-upload/v0.0.2/step.yml (hunk 1 above)
Comment (kawaiseb) [reply steps/wetransfer-upload/v0.0.2/step.yml]: Ok

**Comment by kawaiseb** [comment]
Hum... ok i will try it from git... i am sure that the nodejs script is working.. but i don't know exactely how to pass parameter to my script. Let me test more...

**Pull request closed without merging**